wiki/Tournaments/GBC/GBC_2024_Autumn/en.md

```diff
@@ -57,12 +96,244 @@ The GB Cup 2024 Autumn is run by Team GB and various community members.
+| D12 | **Photon Worshippers** | ::{ flag=US }:: **[moths](https://osu.ppy.sh/users/17244618)**, ::{ flag=BR }:: [Netroo](https://osu.ppy.sh/users/13769101)**, ::{ flag=MX }:: [happergamer](https://osu.ppy.sh/users/15513319)**, ::{ flag=CA }:: [LazyGhost14](https://osu.ppy.sh/users/13185425)**, ::{ flag=JP }:: [fanfale](https://osu.ppy.sh/users/32830556) |
+| D13 | **bol** | ::{ flag=PH }:: **[Silicosis 2](https://osu.ppy.sh/users/18560307)**, ::{ flag=MY }:: [JayLye](https://osu.ppy.sh/users/14892447)**, ::{ flag=CN }:: [[Crz]raber](https://osu.ppy.sh/users/6753592)**, ::{ flag=ID }:: [Yangga](https://osu.ppy.sh/users/12143152)**, ::{ flag=HK }:: [ERDFS](https://osu.ppy.sh/users/28707779) |
+| D14 | **The Vicious Cycle Of Cat** | ::{ flag=TH }:: **[AomDCO](https://osu.ppy.sh/users/21072569)**, ::{ flag=PL }:: [bagjettka](https://osu.ppy.sh/users/18338179)**, ::{ flag=TH }:: [panny](https://osu.ppy.sh/users/20179559)**, ::{ flag=SG }:: [AdamAckerville](https://osu.ppy.sh/users/12297375)**, ::{ flag=PH }:: [BronyaZaychik_](https://osu.ppy.sh/users/19690810) |
+| D15 | **Mania Rangers Dinoforce** | ::{ flag=KR }:: **[Selitation](https://osu.ppy.sh/users/25311011)**, ::{ flag=PH }:: [lyvet](https://osu.ppy.sh/users/18121419)**, ::{ flag=ES }:: [Alen-](https://osu.ppy.sh/users/9552883)**, ::{ flag=DZ }:: [aeko](https://osu.ppy.sh/users/16665306)**, ::{ flag=PH }:: [PandaChee](https://osu.ppy.sh/users/8072250) |
+| D16 | **okay²** | ::{ flag=TH }:: **[parponger](https://osu.ppy.sh/users/28098890)**, ::{ flag=KR }:: [gaesol](https://osu.ppy.sh/users/12452131)**, ::{ flag=RO }:: [[LS]Khu](https://osu.ppy.sh/users/12712916)**, ::{ flag=CA }:: [[LS]Nemesis](https://osu.ppy.sh/users/20544648)**, ::{ flag=JP }:: [yuukituk](https://osu.ppy.sh/users/11998955) |
+
+## Podium
+
+This competition has come to an end and resulted in the following podium:
+
+### QQ division
+
+| Placing | Player |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Q17: 雾雨魔法小萝莉 |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | Q5: 我已进入天空之城 |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | Q3: 漏勺挖洞, Q24: Make PLANA Great Againn |
+
+### Discord division
+
+| Placing | Player |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | D16: okay² |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | D12: Photon Worshippers |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | D13: bol |
 
 ## Mappools
 
+### Knockout Week 2 Mappool
+
+**[Download the mappack here (172MB)](https://drive.google.com/file/d/1iz317E_XpGKWZz7uF66TkWCP-clCMTV8/view?usp=sharing)**
+
+- **RC**
```

Comment:
please follow the [template](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/TEMPLATE.md?plain=1#L112-L115)

Reply:
fixed

Reply:
i meant the mappool lists <.<

should look like [this](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/GBC/GBC_2024_Spring/en.md?plain=1#L129-L147), since we don't use acronyms:
![image](https://github.com/user-attachments/assets/72de858a-841b-4056-820b-efe12a1d1b2d)

Reply:
fixed